src/modules/endpoint/sagas/loadEvents.js

```diff
@@ -19,8 +18,10 @@ function * fetchEvents (city: string, language: string): Saga<Array<EventModel>>
 function * loadEvents (
   city: string, language: string, dataContainer: DataContainer, shouldUpdate: boolean
 ): Saga<FetchMapType> {
-  const context = new DatabaseContext(city, language)
-  const eventsAvailable = yield call({ context: dataContainer, fn: dataContainer.eventsAvailable }, context)
+  const eventsAvailable = yield call(
+    (city: string, language: string) => dataContainer.eventsAvailable(city, language),
```

Comment:
Same as above

Reply:
Events require the language.

Reply:
I meant the ` yield call(dataContainer.categoriesAvailable, city, language)` proposal, but apparently the same answer.